packages/gatsby/src/redux/index.ts

```diff
@@ -62,7 +62,9 @@ export const readState = (): IGatsbyState => {
 }
 
 export interface IMultiDispatch {
-  <T extends ActionsUnion>(action: Array<T>): Array<T>
+  <T extends ActionsUnion | ThunkAction<any, IGatsbyState, any, ActionsUnion>>(
+    action: Array<T>
+  ): Array<T>
```

Comment:
our store wasn't accepting thunks, it didn't came up earlier as all our thunks (`createJobV2` and `createNode` where in `.js` file and not `.ts`)